MITLibraries/.github
# Add shared python workflows

#### Why these changes are being introduced:
First attempt at making shared Github Actions workflows for our Python repositories.

#### How this addresses that need:
* Adds a python shared workflow for running tests and coverage reporting
* Adds a python shared workflow for running linters
* Updates REAMDE with Python shared workflow instructions

#### How can a reviewer verify these changes?
See (or re-run) the oai-pmh-harvester actions using these shared workflows here: https://github.com/MITLibraries/oai-pmh-harvester/tree/shared-github-workflow.
